Console/Command/NostoAccountRemoveCommand.php

```diff
@@ -0,0 +1,195 @@
+ * ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
+ * (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS
+ * SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ *
+ * @author Nosto Solutions Ltd <contact@nosto.com>
+ * @copyright 2017 Nosto Solutions Ltd
+ * @license http://opensource.org/licenses/BSD-3-Clause BSD 3-Clause
+ *
+ */
+
+namespace Nosto\Tagging\Console\Command;
+
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Style\SymfonyStyle;
+use Nosto\Tagging\Helper\Account as NostoAccountHelper;
+use Nosto\Tagging\Helper\Scope as NostoScopeHelper;
+use Magento\Store\Model\ScopeInterface;
+use Magento\Store\Model\Store;
+use Magento\Framework\App\Config\Storage\WriterInterface;
+use Nosto\Tagging\Helper\Cache as NostoHelperCache;
+
+class NostoAccountRemoveCommand extends Command
+{
+
+    const SCOPE_CODE = 'scope-code';
+
+    /*
+     * @var NostoAccountHelper
+     */
+    private $nostoAccountHelper;
+
+    /**
+     * @var NostoHelperScope
+     */
+    private $nostoScopeHelper;
+
+    /**
+     * @var bool
+     */
+    private $isInteractive = true;
+
+    /**
+     * @var WriterInterface
+     */
+    private $config;
+
+    /**
+     * @var NostoHelperCache
+     */
+    private $nostoHelperCache;
+
+    /**
+     * NostoAccountRemoveCommand constructor.
+     * @param NostoAccountHelper $nostoAccountHelper
+     * @param NostoScopeHelper $nostoScopeHelper
+     * @param WriterInterface $appConfig
+     * @param NostoHelperCache $nostoHelperCache
+     */
+    public function __construct(
+        NostoAccountHelper $nostoAccountHelper,
+        NostoScopeHelper $nostoScopeHelper,
+        WriterInterface $appConfig,
+        NostoHelperCache $nostoHelperCache
+    ) {
+        $this->nostoAccountHelper = $nostoAccountHelper;
+        $this->nostoScopeHelper  = $nostoScopeHelper;
+        $this->config = $appConfig;
+        $this->nostoHelperCache = $nostoHelperCache;
+        parent::__construct();
+    }
+
+    /**
+     * Configure the command and the arguments
+     */
+    public function configure()
+    {
+        $this->setName('nosto:account:remove')
+            ->setDescription('Remove Nosto Account Via CLI')
+            ->addOption(
+                self::SCOPE_CODE,
+                null,
+                InputOption::VALUE_REQUIRED,
+                'Store view code'
+            );
+        parent::configure();
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function execute(InputInterface $input, OutputInterface $output)
+    {
+        $this->isInteractive = !$input->getOption('no-interaction');
+        $io = new SymfonyStyle($input, $output);
+        $scopeCode = $input->getOption(self::SCOPE_CODE) ?:
```

Comment:
More a readability thing. Just use an `if` instead of a ternary. You use ifs everywhere.